bika/lims/browser/js/bika.lims.analysisrequest.add_by_col.js

```diff
@@ -2598,7 +2598,7 @@ function AnalysisRequestAddByCol() {
                             var destination = window.location.href.split("/portal_factory")[0]
                             var ars = data['stickers']
                             var stickertemplate = data['stickertemplate']
-                            var q = "/sticker?autoprint=1&template=" + stickertemplate + "&items=" + ars.join(",")
+                            var q = "/sticker?autoprint=1&ctype=AnalysisRequest&template=" + stickertemplate + "&items=" + ars.join(",") + ''
```

Comment:
As per https://github.com/naralabs/bika.lims/pull/36/files#r106140430 adding a `ctype` param in the request is not required at all